app/upgrades/v0.1.11.go

```diff
@@ -2,30 +2,26 @@ package upgrades
 
 import (
 	"context"
-	"fmt"
-	"time"
 
+	cosmoslog "cosmossdk.io/log"
 	storetypes "cosmossdk.io/store/types"
 	upgradetypes "cosmossdk.io/x/upgrade/types"
 	cosmostypes "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
-	"github.com/cosmos/cosmos-sdk/x/authz"
 
 	"github.com/pokt-network/poktroll/app/keepers"
 )
 
-// TODO_MAINNET_MIGRATION(@bryanchriswhite): Update the upgrade version numbers, diffs, hashes, etc. and rename this file.
-
 const (
-	Upgrade_0_1_x_PlanName = "v0.1.x"
+	Upgrade_0_1_11_PlanName = "v0.1.11"
 )
 
-// Upgrade_0_1_x handles the upgrade to release `v0.1.x`.
+// Upgrade_0_1_11 handles the upgrade to release `v0.1.11`.
 // This is planned to be issued on both Pocket Network's Shannon Alpha, Beta TestNets as well as MainNet.
 // It is an upgrade intended to reduce the memory footprint when iterating over Suppliers and Applications.
```

Comment:
This is not the intent of the current upgrade

Reply:
:sweat_smile:  Thank you sir!